tasks/sep/004-fpac-upgrade/input-template.json

```diff
@@ -0,0 +1,72 @@
+{
+  "version": "1.0",
+  "chainId": "5",
+  "createdAt": 1707763323,
+  "meta": {
+    "name": "FPAC OptimismPortalProxy upgrade - OP Sepolia Devnet 0",
+    "description": "Upgrades the OptimismPortalProxy's implementation to the OptimismPortal2 contract.",
+    "txBuilderVersion": "1.16.3"
+  },
+  "transactions": [
+    {
+      "to": "<op-sepolia-devnet-0-proxy-admin-address>",
+      "value": "0",
+      "data": "<upgrade-and-call-data>",
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address",
+            "name": "_proxy",
+            "type": "address"
+          },
+          {
+            "internalType": "address",
+            "name": "_implementation",
+            "type": "address"
+          },
+          {
+            "internalType": "bytes",
+            "name": "_data",
+            "type": "bytes"
+          }
+        ]
+      },
+      "contractInputsValues": {
+        "_proxy": "<op-sepolia-portal-proxy-address>",
+        "_implementation": "<op-sepolia-devnet-0-storage-setter-address>",
+        "_data": "<storage-set-data>"
+      }
+    },
+    {
```

Comment:
any reason to commit the input template? not sure if it seems useful

Reply:
Yeah - when we do this on another network, we'll fill out the template w/ diff addresses. i.e. for the Sepolia Testnet upgrade.